addons/mk6mortar/functions/fnc_canLoadMagazine.sqf

```diff
@@ -0,0 +1,54 @@
+/*
+ * Author: Grey
+ * Checks whether magazine can be loaded into static weapon
+ *
+ * Arguments:
+ * 0: static <OBJECT>
+ * 1: unit <OBJECT>
+ * 2: magazineClassOptional <OPTIONAL><STRING>
+ *
+ * Return Value:
+ * canLoadMagazine <BOOL>
+ *
+ * Example:
+ * [_target,_player,"ACE_1Rnd_82mm_Mo_HE"] call ace_mk6mortar_fnc_canLoadMagazine
+ *
+ * Public: Yes
+ */
+
+#include "script_component.hpp"
+
+params ["_static","_unit",["_magazineClassOptional","",[""]]];
+private ["_canLoadMagazine","_currentMagazine","_weapon","_listOfMagNames",
+    "_hasCompatibleMagazine","_count"];
+
+if !(alive _static && {GVAR(useAmmoHandling)}) exitWith {false};
```

Comment:
this should be 
`(!alive _static && {!GVAR(useAmmoHandling)})`
else the lasy elevation have 0 resasons